Application/EdFi.Ods.SwaggerUI/appsettings.json

```diff
@@ -6,6 +6,18 @@
       "ClientSecret": ""
     }
   },
+  "Years": [
+    {
+      "Year": "2020"
+    },
+    {
+      "Year": "2021",
+      "IsDefault": true
+    },
+    {
+      "Year": "2022"
+    }
+  ],
```

Comment:
Would it be a good idea and remove this so we can just use the defaults? that way we don't have to update this every year. cc @vimayya

Reply:
Done.

Reply:
sure. I will add it to documentation.